### PR TITLE
Prevent windows web doctor from launching chrome

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Due to https://bugs.chromium.org/p/chromium/issues/detail?id=158372, asking chrome for its version on windows will cause an instance to be launched. look in the registry instead

### CODE REVIEW

**Comment by jonahwilliams** [comment]
WIP because I haven't run it on windows yet!

**Comment by miyoyo** [comment]
This powershell one-liner prints the version of chrome installed, pretty sure it ain't perfect but it works
```powershell
Get-ItemProperty HKLM:\Software\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\* | Where-Object DisplayName -eq "Google Chrome" | Select-Object -expand Version
```
Returns
`74.0.3729.169`

**Review by dnfield — APPROVED** [review]
This makes me sad but LGTM.

File: packages/flutter_tools/lib/src/web/web_device.dart [hunk 1]
```diff
@@ -82,15 +85,29 @@ class WebDevice extends Device {
...
   @override
   Future<String> get sdkNameAndVersion async {
-    final String chrome = findChromeExecutable();
-    final ProcessResult result = await processManager.run(<String>[
-      chrome,
-      '--version',
-    ]);
-    if (result.exitCode == 0) {
-      return result.stdout;
+    // See https://bugs.chromium.org/p/chromium/issues/detail?id=158372
```
Comment (dnfield) [inline packages/flutter_tools/lib/src/web/web_device.dart]: Maybe make this a `TODO` to fix this when that bug is resolved?

Comment (dnfield) [reply packages/flutter_tools/lib/src/web/web_device.dart]: Nevermind it's wontfix.

**Review by yjbanov — APPROVED** [review]
lgtm

**Pull request merged**